bundles/org.openhab.binding.wled/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -41,37 +28,77 @@
 			<parameter name="pollTime" type="integer" required="true" min="1" unit="s">
-				<label>Poll States</label>
+				<label>Poll Time</label>
 				<description>Time in seconds of how often to fetch the state of the LEDs.</description>
-				<default>10</default>
-			</parameter>
-			<parameter name="segmentIndex" type="integer" required="true" min="0">
-				<label>Segment Index</label>
-				<description>Leave this as 0 if you are not using segments, otherwise set this to the segment index number that you
-					wish to control.</description>
-				<default>0</default>
+				<default>5</default>
 			</parameter>
 			<parameter name="saturationThreshold" type="integer" required="true" min="0" max="99">
 				<label>Saturation Threshold</label>
 				<description>This feature allows you to specify a number that if the saturation drops below, will trigger white.
 				</description>
+				<advanced>true</advanced>
 				<default>0</default>
 			</parameter>
 			<parameter name="sortEffects" type="boolean">
 				<label>Sort Effects</label>
 				<description>If set, will sort the state options of the effects channel alphabetically while keeping the first
 					option (Solid) at the top.</description>
-				<default>false</default>
+				<advanced>true</advanced>
+				<default>true</default>
 			</parameter>
 			<parameter name="sortPalettes" type="boolean">
 				<label>Sort Palettes</label>
 				<description>If set, will sort the state options of the palettes channel alphabetically while keeping the first
 					option (Default) at the top.</description>
-				<default>false</default>
+				<advanced>true</advanced>
+				<default>true</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="segment">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="json"/>
+		</supported-bridge-type-refs>
+		<label>WLED Segment</label>
+		<description>Controls an entire LED strip, or a section of the strip if the LED string is split into multiple
+			segments.</description>
+		<category>ColorLight</category>
+		<channels>
+			<channel id="masterControls" typeId="masterControls"/>
+			<channel id="segmentBrightness" typeId="segmentBrightness"/>
+			<channel id="primaryColor" typeId="primaryColor"/>
+			<channel id="primaryWhite" typeId="primaryWhite"/>
+			<channel id="secondaryColor" typeId="secondaryColor"/>
+			<channel id="secondaryWhite" typeId="secondaryWhite"/>
+			<channel id="tertiaryColor" typeId="tertiaryColor"/>
+			<channel id="tertiaryWhite" typeId="tertiaryWhite"/>
+			<channel id="palettes" typeId="palettes"/>
+			<channel id="fx" typeId="fx"/>
+			<channel id="speed" typeId="speed"/>
+			<channel id="intensity" typeId="intensity"/>
+			<channel id="mirror" typeId="mirror"/>
+			<channel id="reverse" typeId="reverse"/>
+			<channel id="grouping" typeId="grouping"/>
+			<channel id="spacing" typeId="spacing"/>
+		</channels>
+		<config-description>
+			<parameter name="segmentIndex" type="integer" required="true" min="0">
```

Comment:
When you provide a default value, this is not necessary to mark the parameter as required.